Library/test-helpers/lib.sh

```diff
@@ -1950,11 +2198,15 @@ Returns 0.
 
 =cut
 
-limeconStopAgent() {
+limeconStop() {
 
         local NAMES=$1
 
         podman ps -a --format "{{.Names}}" | grep -e "^${NAMES}\$" | xargs podman stop | xargs podman rm
+
+        if [[ "agent_container" = *"$NAMES"* ]];then
+            chmod o-rw /dev/tpmrm0
```

Comment:
We cannot do that here because there could be other agents still running (with multiple agents scenario). Moreover `agent_container` is just a specific name used in this particular test, not a generic one. Either do this in a setup&cleanup phases in a container test itself (my choice for now) or in the `limeBackupConfig` and `limeRestoreConfig` or we could eventually introduce new `limeconBackupConfig` and `limeconBackupConfig` or something similar.